src/ipyforcegraph/behaviors/scales.py

```diff
@@ -75,6 +75,8 @@ class Scheme(enum.Enum):
         help=("the ``[min, max]`` to map to the scale's colors"),
     ).tag(sync=True)
 
+    tint: float = T.Float(0).tag(sync=True)
```

Comment:
Would a good help for this be:  `help="brighten (``tint>0``) or darken (``tint<0``) the colors",` ?

Reply:
let's see if it's even any good, but yes, needs docs